go/utils/websocket.go

```diff
@@ -9,10 +9,10 @@ import (
 // ConnectWithRetry will attempt to connect a websocket to the given url
 // retrying if needed with the given delay (milliseconds) between calls
 func ConnectWithRetry(url string, delay int) (socket *websocket.Conn, err error) {
-	Retry(delay, func() bool {
+	Retry(delay, func(updatedDelay int) bool {
 		socket, _, err = websocket.DefaultDialer.Dial(url, nil)
 		if err != nil {
-			fmt.Printf("Unable to connect (%s). Retrying...\n", err)
+			fmt.Printf("Unable to connect (%s). Retrying after %v ms...\n", err, updatedDelay*2)
```

Comment:
I don't believe this log is accurate, as this fn is passed the initialDelay and the not the nextDelay

Reply:
yes, but the `initialDelay` is updated in the `Retry` method's `for` loop, the log is anticipating the doubling of the delay, hence the `updatedDelay * 2`